roles/analysis/tasks/analysis-c2r.yml

```diff
@@ -0,0 +1,40 @@
+---
+# TODO: Install convert2rhel for different OS?
+- name: Install packages for pre-conversion analysis on CentOS 7
+  ansible.builtin.package:
+    name: "{{ analysis_packages_centos7 }}"
+    # enablerepo: "{{ analysis_repos_centos7 }}" TODO?
```

Comment:
We have a specific repository for this, but I think it won't be able to enable through normal subscription-manager... We will need to rely on the common role for that, I believe. (its mapped there the repos and how to install the package.)

Reply:
Removed